store/todos.js

```diff
@@ -20,6 +20,9 @@ export const getters = {
   isTodoCreating(state) {
     return state.isCreating
   },
+  getEditableTodo(state) {
+    return state.editableTodo
+  },
```

Comment:
while this works, there can be cases we want to edit multiple todo at once maybe?

Reply:
noted.